skills/multimodal.md

```
@@ -0,0 +1,345 @@
+      description: "Login button has 2.1:1 contrast ratio (needs 4.5:1)"
+      suggestion: "Change button color from #999 to #666"
+
+    - severity: medium
+      type: layout
+      description: "Form not centered on mobile viewport"
+      suggestion: "Add flexbox centering to .login-container"
+```
+
+### Error Screenshot Analysis
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)